content/en/boilerplates/revision-tags-preamble.md

```diff
@@ -0,0 +1,11 @@
+---
+---
+Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone.
+[Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are a solution to this.
```

Comment:
These links don't give a lot of information, I would suggest pointing to this blog instead: https://deploy-preview-10112--preliminary-istio.netlify.app/latest/blog/2021/revision-tags/

Reply:
Updated